bedrock/pocket/views.py

```diff
@@ -13,3 +21,33 @@ def server_error_view(request, template_name="pocket/500.html"):
 def page_not_found_view(request, exception=None, template_name="pocket/404.html"):
     """404 error handler that runs context processors."""
     return l10n_utils.render(request, template_name, ftl_files=["pocket/404"], status=404)
+
+
+@require_POST
+def newsletter_subscribe(request):
+    try:
+        data = json.loads(request.body)
+    except Exception:
+        return JsonResponse({"error": "Error parsing JSON data"}, status=400)
+
+    external_id = request.COOKIES.get(settings.BRAZE_POCKET_COOKIE_NAME, None)
```

Comment:
Can this result in an empty string instead of `None` if the cookie exists but is empty? That's why I had it `or None` before. Though empty string is probably okay in context as well being falsy as it is.

Reply:
Ah yeah - i was noodling with that and should have left it as it was. Will revert as I think your way is safer